docs/reference/release-policy.md

```diff
@@ -1,7 +1,22 @@
-# Supported tracks
+# Release policy
+
+Our release policy includes two kinds of releases, short-term releases and long-term support (LTS) releases.
+
+We release every six months, same as Ubuntu, and our LTS releases coincide with Ubuntu.
```

Comment:
You could link to Ubuntu's release cycle documentation: https://ubuntu.com/about/release-cycle (e.g., "...coincide with [Ubuntu's LTS release cadence](https://ubuntu.com/about/release-cycle)."

I don't know if that's necessary for your users / how important it is to them that you line up with Ubuntu releases. Sharing the doc just in case